frontend/src/components/LandingPage/LandingPage.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react'
-//Import components that you create here
+import { MyNavbar } from "../MyNavbar/MyNavbar";
 
 function LandingPage(){
     return(
-        //Use components that you import here
-        <></>
+        <>
+            <MyNavbar />
```

Comment:
I think since the navbar isnt exclusive to the LandingPage that we should move it to App.js and import there instead. What do you think?

Reply:
Ill provide more context here. I think it might be better to render the navbar inside of app.js and not in the landing page as it is here. That way when we navigate away from the landing page, the navbar will still be seen.

Reply:
I think the navbar should stay as a separate component because there is quite a bit of code. I think it will make the code for landing page easier to read if navbar is a separate component.